app/src/main/kotlin/com/passwordstore/android/ui/activity/FirstRunActivity.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2014-2019 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+package com.passwordstore.android.ui.activity
+
+import android.os.Bundle
+import android.view.ViewGroup
+import com.passwordstore.android.databinding.ActivityFirstRunBinding
+import com.passwordstore.android.ui.EdgeToEdge
+
+class FirstRunActivity : BaseActivity() {
+
+    private lateinit var binding: ActivityFirstRunBinding
```

Comment:
Don't make this a class variable when we're not using it outside `onCreate`

Reply:
:+1: Okay